styles.css

```diff
@@ -10,4 +10,8 @@ p {
 
 .header {
   background: goldenrod;
+
+}
+.site-footer {
```

Comment:
Is `site-footer` the best name for this? Considering that header is called `site-header`?